client/views/session.tpl

```diff
@@ -0,0 +1,17 @@
+<p class="clearfix">
+{{#if current}}
+	<strong>{{agent}}</strong>
+	<a href="https://ipinfo.io/{{ip}}" target="_blank" rel="noopener">{{ip}}</a>
+{{else}}
+	<button class="btn pull-right remove-session" data-token="{{token}}">Disconnect</button>
+
+	<strong>{{agent}}</strong>
+	<a href="https://ipinfo.io/{{ip}}" target="_blank" rel="noopener">{{ip}}</a>
+	<br>
+	{{#if active}}
+		<b>Currently active</b>
```

Comment:
I would use `em` instead of `b` to contrast it with the user agent line but again nbd if you don't want to.